mobile/test/common/integration/client_integration_test.cc

```diff
@@ -46,13 +46,14 @@ class TestKeyValueStore : public Envoy::Platform::KeyValueStore {
   std::string value_;
 };
 
-class ClientIntegrationTest : public BaseClientIntegrationTest,
-                              public testing::TestWithParam<Network::Address::IpVersion> {
+class ClientIntegrationTest
+    : public BaseClientIntegrationTest,
+      public testing::TestWithParam<std::tuple<Network::Address::IpVersion, Http::CodecType>> {
 public:
   static void SetUpTestCase() { test_key_value_store_ = std::make_shared<TestKeyValueStore>(); }
   static void TearDownTestCase() { test_key_value_store_.reset(); }
 
-  ClientIntegrationTest() : BaseClientIntegrationTest(/*ip_version=*/GetParam()) {
+  ClientIntegrationTest() : BaseClientIntegrationTest(/*ip_version=*/std::get<0>(GetParam())) {
```

Comment:
nit: a couple helper functions might make it easier to read:

IpVersion getIpVersion() { return std::get<0>(GetParam()); }
CodecType getCodecType() { return std::get<1>(GetParam()); }

Reply:
sgtm!